bigtable/client/client_options.h

```diff
@@ -92,6 +92,27 @@ class ClientOptions {
     channel_arguments_.SetGrpclbFallbackTimeout(fallback_timeout);
   }
 
+  /**
+   * Set the grpclb fallback timeout (std::chrono::duration<> duration) for the
+   * channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetGrpclbFallbackTimeout()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  template <typename Rep, typename Period>
+  void SetGrpclbFallbackTimeout(
+      std::chrono::duration<Rep, Period> fallback_timeout) {
+    int fallback_timeout_ms = 0;
```

Comment:
Please move the variable definition to the point where it is initialized (line 111).